app/Http/Controllers/Traits/ListsSites.php

```diff
@@ -51,7 +60,20 @@ public function getSites($sites)
                     });
                 });
             })
-            ->orderBy($request->order_by ?? 'created_at', $request->order_dir ?? 'desc');
+            ->when($request->site_type, function ($query) use ($request) {
+                switch ($request->site_type)
+                {
+                    case 'all':
+                        $query->withShared($request->user())->orWhere('user_id', $request->user()->id);
```

Comment:
Whenever adding orWhere, always group the query in a `where(function($query) ...)`. Without grouping this could be a dangerous query. (if you need more info let me know)

```suggestion
                        $query->where(function($query) use ($request) {
                            $query->withShared($request->user());
                            $query->orWhere('user_id', $request->user()->id);
                        });
```